sdccc/src/main/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ConditionalPreconditions.java

```diff
@@ -186,6 +188,42 @@ private static boolean descriptionUpdateManipulation(final Injector injector) {
         return ResponseTypes.Result.RESULT_SUCCESS.equals(manipulationResult);
     }
 
+    private static boolean descriptionUpdateWithParentChildRelationshipManipulation(final Injector injector, final Logger log) {
+        Pair<String, String> descriptorHandles = findFirstDescriptorHandleWithChildren(injector, log);
+        if (descriptorHandles == null) {
+            log.error("Could not trigger a Descriptor Update with Parent Child Relationship as there are no " +
+                    "Descriptors with Children in the Mdib.");
+            return false;
+        }
+        String parentDescriptorHandle = descriptorHandles.getLeft();
+        String childDescriptorHandle = descriptorHandles.getRight();
+        final var manipulations = injector.getInstance(Manipulations.class);
+
+        final ResponseTypes.Result manipulationResult = manipulations.triggerDescriptorUpdate(List.of(childDescriptorHandle, parentDescriptorHandle));
+
+        return ResponseTypes.Result.RESULT_SUCCESS.equals(manipulationResult);
+    }
+
+    private static Pair<String, String> findFirstDescriptorHandleWithChildren(Injector injector, Logger log) {
```

Comment:
Checkstyle will complain here, please make the parameters `final`.

Reply:
Done.